src/utils/transactions.js

```diff
@@ -13,6 +13,14 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import elements from 'lisk-elements';
+import { ValidationError } from './error';
 
-export default elements.transaction;
```

Comment:
Uhhh, were we using this before?

Reply:
yes we were using this before, but not anymore